src/Sarif.Viewer.VisualStudio/SarifViewerPackage.cs

```diff
@@ -67,11 +58,26 @@ public static SarifToolWindow SarifToolWindow
         {
             get
             {
-                SarifViewerPackage package = SarifViewerPackage.ServiceProvider as SarifViewerPackage;
-
-                SarifToolWindow toolWindow = package?.FindToolWindow(typeof(SarifToolWindow), 0, true) as SarifToolWindow;
-
-                return toolWindow;
+                ThreadHelper.ThrowIfNotOnUIThread();
+                IVsPackage package;
```

Comment:
>IVsPackage [](start = 16, length = 10)

Nit/suggestion: Move declaration down to right above point of use (or even collapse it into the `IsPackageLoaded` call (`out IVsPackage package`).